docs/BasisTheoryElements.md

```diff
@@ -1,17 +1,17 @@
 # BasisTheoryElements
 
-This class contains functions to pass Element references when calling the Basis Theory API.
+This class contains functions to interact with the Basis Theory API using element references.
 
 ## Instantiation
 
 A new instance of the `BasisTheoryElements` service can be instantiated and configured using the
-builder pattern, for example:
+builder pattern:
 
 ```kotlin
 val bt = BasisTheoryElements.builder()
-    .apiUrl("https://api.basistheory.com")
+    .apiUrl("https://api.basistheory.com") // optional
```

Comment:
I wouldn't include this here since by default it is prod and they shouldn't be switching it to anything else

Reply:
We won't show it on the main top level readme example in case someone copies that, but I want to show all the configuration options available on some docs page, and this is that place - I'll be moving this page into the new docs site when we port some of this over there and take it out of the readme.